src/vs/editor/common/config/editorOptions.ts

```diff
@@ -5602,6 +5608,10 @@ export const EditorOptions = {
 		EditorOption.occurrencesHighlight, 'occurrencesHighlight', true,
 		{ description: nls.localize('occurrencesHighlight', "Controls whether the editor should highlight semantic symbol occurrences.") }
 	)),
+	multiDocumentOccurrencesHighlight: register(new EditorBooleanOption(
```

Comment:
Should this be a new setting or should we extends the existing `occurrencesHighlight` setting? E.g extends its value set to `acrossDocuments` (we have done something similar for word based suggestions which can be a boolean or one of three values)